eogrow/pipelines/export_maps.py

```diff
@@ -98,29 +98,30 @@ def build_workflow(self) -> EOWorkflow:
 
         return EOWorkflow(linearly_connect_tasks(*task_list))
 
-    def create_maps(self, feature_name: str, map_name: str, patch_names: List[str], utm_split: bool = True) -> None:
+    def create_maps(self, feature_name: str, patch_names: List[str]) -> None:
         """A method which creates a joined GeoTIFF maps from a list of exported GeoTIFFs"""
         if not patch_names:
             raise ValueError("Cannot create map with an empty list of EOPatch names")
 
         folder = fs.path.join(self.storage.get_folder(self.config.output_folder_key), feature_name)
 
-        if utm_split:  # TODO: this can be parallelized but _create_single_map would have to become static
-            crs_eopatch_dict = self.eopatch_manager.split_by_utm(patch_names)
+        crs_eopatch_dict = self.eopatch_manager.split_by_utm(patch_names)
 
-            for crs, eopatch_list in crs_eopatch_dict.items():
-                utm_map_name = f"utm{crs.epsg}_{map_name}"
-                self._create_single_map(folder, eopatch_list, utm_map_name)
-        else:
-            self._create_single_map(folder, patch_names, map_name)
+        for crs, eopatch_list in crs_eopatch_dict.items():
```

Comment:
Perhaps we can leave the TODO about parallelization?

Reply:
added